shorten_path_for_notational_fzf.py

```diff
@@ -65,20 +65,31 @@ def color(line, color):
 
 def process_line(line: str) -> str:
     # Expected format is colon separated `name:line number:contents`
+
+    # Windows paths may contain a colon, e.g. C:\Windows\ which messes up the split
+    # splitdrive(string) results in the following:
+    #   Windows drive letter, e.g. C:\Windows\Folder\Foo.txt -> ('C', '\Windows\Folder\Foo.txt')
+    #   Windows UNC path, e.g. \\Server\Share\Folder\Foo.txt -> ('\\Server\Share', '\Folder\Foo.txt')
+    #   *nix, e.g. /any/path/to/file.txt -> ('', '/any/path/to/file.txt')
+    #  Replace the : with a _ because the : will cause problems with FZF's delimiters, too.
+    drive, remainder = splitdrive(line)
+    line = drive.replace(':', '_') + remainder
```

Comment:
Since the C drive is used so often, it may be worth just dropping the drive entirely from the shortened name.

Reply:
Done!